Zadanie 11.1 Pierwsza klasa/js/scripts.js

```diff
@@ -0,0 +1,33 @@
+// scripts.js
+var iPhone6S = new Phone("Apple", 2250, "silver");
+var samsungGalaxyS6 = new Phone("Samsung", 2100, "black");
+var onePlusOne = new Phone("One Plus", 1700, "gold");
+
+function Phone (brand, price, color) {
+    this.brand = brand;
+    this.price = price;
+    this.color = color;
+}
+
+Phone.prototype.printInfo = function () {
+    console.log("The phone brand is " + this.brand + " price is " + this.price + " and the color is " + this.color + ".");
+}
+
+iPhone6S.printInfo();
+samsungGalaxyS6.printInfo();
+onePlusOne.printInfo();
+
+// diffrent method
+
+var Phone = {
```

Comment:
konwencja jest taka, ze z wielkiej litery nazywa sie funkcje, ktore sa "konstruktorami" przeznaczonymi do wywolywania z `new`. Dlatego jesli to jest zwykly obiekt to nazwij go z malej litery i zmien nazwe, bo teraz nadpisujesz `Phone` i nie da sie go juz uzyc jako konstruktora: https://scr.hu/yzyKzY